src/renderer/screens/manager/AppsList/index.js

```diff
@@ -38,14 +52,29 @@ const AppsList = ({ deviceInfo, result, exec, t }: Props) => {
         plan={plan}
         progress={progress}
       />
-      <AppList
-        deviceInfo={deviceInfo}
-        state={filteredState}
+      <AppsListContext.Provider value={{ setAppInstallDep, setAppUninstallDep }}>
```

Comment:
this will re-trigger consumer render each time this component render. we need to memoize the value we send to Provider.

more generally, i'm not sure to understand why we need a context here. Can't we send this by props?